tests/dummy/app/templates/docs/buttons.md

```diff
@@ -0,0 +1,40 @@
+# Buttons
+
+`ember-caluma` provides buttons for mutating the state of work items.
+
+## Usage
+
+To use these component call them with the required parameters.
+
+### WorkItemButton
+
+The text of the button is per default the mutation name, it can be overwritten with either the `label` parameter or using the component in block form.
+
+```hbs
+<WorkItemButton @mutation="complete" @workItemId={{this.workItemId}}/>
+
+<WorkItemButton
+  @label="Lorem Ipsum"
+  @mutation="complete"
+  @workItemId={{this.workItemId}}
+/>
+
+<WorkItemButton @mutation="complete" @workItemId={{this.workItemId}}>
+  Lorem Ipsum
+</WorkItemButton>
+```
+
+ The WorkItemButton component expects 2 parameters:
+
+ - `mutation` The state mutation to be used either `complete`, `skip` or `cancel`.
+ - `workItemId` Id of the work item which should be mutated.
+ - `label` (Optional) Overwrites the button text.
+
+ Additionally there are many optional parameters for the underlying UkButton component, refer to the UkButton component documentation to see what exactly they do:
```

Comment:
Link the docs. Other people might not know that `UkButton` comes from `ember-uikit` ...

Reply:
Added a link